app-backend/db/src/main/resources/migrations/V11__Add_stac_export_table.sql

```diff
@@ -0,0 +1,15 @@
+-- Add a table for label stac export
+
+CREATE TABLE public.stac_exports (
```

Comment:
I think we need to add indices for at least the following the following:

 - owner
 - created_by
 - export_status